examples/projections/nongeo/cartesian_linear.py

```diff
@@ -4,18 +4,16 @@
 
 **X**\ *width*\ [/*height*] or **x**\ *x-scale*\ [/*y-scale*]
 
-Give the *width* of the figure and the optional *height*.
-The lower-case version **x** is similar to **X** but expects
-an *x-scale* and an optional *y-scale*.
+Give the *width* of the figure and the optional *height*. The lower-case version
+**x** is similar to **X** but expects an *x-scale* and an optional *y-scale*.
 
-The Cartesian linear projection is primarily designed for regular
-floating point data. To plot geographical data in a linear
-projection, see the upstream GMT documentation
-:gmt-docs:`Geographic coordinates
+The Cartesian linear projection is primarily designed for regular floating point
+data. To plot geographical data in a linear projection, see the upstream GMT
+documentation :gmt-docs:`Geographic coordinates
 <reference/coordinate-transformations.html#geographic-coordinates>`.
```

Comment:
It's better to write the `gmt-docs` role in one line, like:
```
:gmt-docs:`Geographic coordinates <reference/coordinate-transformations.html#geographic-coordinates>`
```

Reply:
Hm. I just tried this in commit https://github.com/GenericMappingTools/pygmt/pull/3473/commits/864579ea81bd5f5b9ed21247e3d0007c9582f35a, but the style checks fail (locally and on GitHub) because of the too long line.

Reply:
OK. please revert the changes.

Reply:
Done in commit https://github.com/GenericMappingTools/pygmt/pull/3473/commits/7de29df786117afd3a1e8567f3c256ceb7aad345.